### ISSUE TITLE
does anybody know whether it's possible to see lies under localhost?

### ISSUE THREAD

**Issue by shailangsa** [abrahamjuliot/creepjs#231]
as the title, does anybody know whether it's possible to see lies under localhost? im testing with a webgl defender chrome extension and although it gives me a randomized fingerprint in the webgl section, it doesnt show under lies 

![image](https://github.com/abrahamjuliot/creepjs/assets/49602578/57759f42-226e-4481-953d-9b64ebedb0e7)

**Comment by abrahamjuliot** [comment]
It's available in the console under the `Loose Fingerprint` section.

**Comment by shailangsa** [comment]
> Loose Fingerprint

i see it now thank you, by the way do you know how running `yarn build:dev` and `yarn watch:dev` helps given that the page can be opened by going into docs/index.html or public/index.html? 

also is there a difference between the index.html inside public/ and the one in docs/ ?

**Comment by abrahamjuliot** [comment]
`yarn build:dev` and `yarn watch:dev` updates code in /public. This is for local development and can be used to run beta testing before releasing to GitHub pages.

docs/ is for publishing to GitHub pages and is the final public release compiled via `yarn build`.

**Comment by shailangsa** [comment]
I see. Is it by design that the lies and loose fp/revisions/session/lies doesnt display on index.html? ( i know they display on console as we found out earlier though)

also how does loose fp differ from lies?

**Comment by abrahamjuliot** [comment]
No, I need to clean that up. It would be nice to see a clean UI regardless of API status codes. Let's keep this issue open. I will set a task to resolve it.

**Comment by abrahamjuliot** [comment]
> how does loose fp differ from lies?

Loose fingerprint contains most of the fingerprints, with some exceptions in the experimental or too unstable category.
Lies are just one area of fingerprints among many.
